typedb/simulation.tql

```diff
@@ -42,15 +44,22 @@ person sub entity,
     owns birth-date,
     owns gender,
     owns address,
+    owns marital-status,
     plays birth-place:child,
     plays residentship:resident,
+    plays nationality:national,
+    plays citizenship:citizen,
     plays marriage:husband,
     plays marriage:wife,
-    plays parentship:parent,
-    plays parentship:child,
+    plays parenthood:parent,
+    plays parenthood:child,
+    plays grandparenthood:grandparent,
+    plays grandparenthood:grandchild,
```

Comment:
I did this purely on the basis that `parenthood` is an actual word that people commonly use, adding `ship` to the end is something we have normalised doing but I think it may confuse readers.

Reply:
Oh really? I thought `parentship` is a real word too! 🤣 https://www.lexico.com/definition/parentship